core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -57,14 +58,27 @@ public void setParameters(Map<String, String> params) throws ParametersException
     ParamsHelper.atLeastOneIsProvided(errorMessagePattern, sourceFile, line);
   }
 
+  /**
+   * Filters JS errors and removes errors ignored because of filters.
+   *
+   * @param data collected JS errors
+   *
+   * @return errors that were not ignored by this filter
+   */
   @Override
-  public Set<JsErrorLog> modifyData(Set<JsErrorLog> data) throws ProcessingException {
-    return FluentIterable.from(data).filter(new Predicate<JsErrorLog>() {
-      @Override
-      public boolean apply(@Nullable JsErrorLog input) {
-        return !shouldFilterOut(input);
-      }
-    }).toSet();
+  public Set<JsErrorLog> modifyData(Set<JsErrorLog> data) {
```

Comment:
Since we have Java 8 maybe we could a `java.util` streams here instead?

Reply:
That is true! I just wanted to introduce as little changes as possible..

Reply:
Exactly:
`return data.stream().filter(input -> !shouldFilterOut(input)).collect(Collectors.toSet());`

Reply:
Yeah! There will be changes in near future that will do something similar, but without filtering ;)